### PR TITLE
login and gamestart

### PR DESCRIPTION
**Pull request by ghost**
Screenshot 2024-04-13 at 5 01 53 PM
Screenshot 2024-04-13 at 5 02 09 PM

- Added Login and GamStart screens 
- need to implement LoginViewModel 
- Created User data class to store user info, but might delete it later

### CODE REVIEW

File: mobile/app/src/main/java/com/cs4520/brainflex/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        lateinit var logInViewModel: LogInViewModel
+        logInViewModel = ViewModelProvider(this).get(LogInViewModel::class.java)
+        setContent {
+            BrainFlexTheme {
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                ) {
+                    AppNavHost(loginVieModel = logInViewModel, navController = rememberNavController())
```
Comment (jacksonlw) [inline mobile/app/src/main/java/com/cs4520/brainflex/MainActivity.kt]:
```suggestion
                    AppNavHost(navController = rememberNavController())
```
Remove line. Stops app from building.

**Comment by jacksonlw** [comment]
Login page input placeholder/label not visible when focused

**Comment by jacksonlw** [comment]
Remove password input. Not collecting passwords on the API.

File: mobile/app/src/main/java/com/cs4520/brainflex/MainActivity.kt (hunk 1 above)
Comment (ghost) [reply mobile/app/src/main/java/com/cs4520/brainflex/MainActivity.kt]: Oops forgot to remove the line--it should work now

**Review by jacksonlw — APPROVED** [review]

**Pull request merged**